encord/http/querier.py

```diff
@@ -220,7 +220,7 @@ def _execute(self, request: Request, retryable=False, enable_logging: bool = Tru
             res = session.send(req, timeout=timeouts)
 
             try:
-                res_json = res.json()
+                res_json = orjson.loads(res.content)
```

Comment:
I tried quite hard to find a library which does streaming parsing but unfortunately it's the python world, so nothing exists.
We'd need to write on of our own, and not in python, and that's not a priority at the moment. ORJSON does well.